Wireframe/index.html

```diff
@@ -5,28 +5,71 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe</title>
     <link rel="stylesheet" href="style.css" />
+    <link
+    href="https://fonts.googleapis.com/css2?family=Mona+Sans:ital,wght@0,200..900;1,200..900&family=PT+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+    rel="stylesheet"/>
   </head>
   <body>
+
+    <div>
     <header>
-      <h1>Wireframe</h1>
+      <h1>Development process</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        In software engineering, a software development process or software development life cycle (SDLC) is a process of planning and managing software development.
       </p>
     </header>
-    <main>
+    <main class="articles">
+      <div class="top-article">
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img 
+         class="article1-img"
+        src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTZObs1r5i2MPoDiIHGEhymP22K_ZZfMBa1Ww&s" 
+        alt="purpose of README file"/>
+        <h2>What is the purpose of a README file?</h2>
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
+          This file serves as the first point of contact for users and developers who want to understand the purpose, setup, and usage of a project. But what is a README.md file exactly?
         </p>
-        <a href="">Read more</a>
+        <a href="https://www.geeksforgeeks.org/what-is-readme-md-file/#what-is-a-readmemd-file"
+        target="_blank">Read more</a>
       </article>
-    </main>
+      </div> 
+      
+      <div class="bottom-articles">
+        <article class="card">
+          <img
+            class="articleimg"
+            src="https://www.freecodecamp.org/news/content/images/2020/08/wireframe-1.gif"
+            alt="wireframe purposes"/>
+            <h3>What is the purpose of a wireframe?</h3>
+    <p>Purpose of Wireframe:
+      Structural Planning.
+      User Flow. 
+      Communication.</p>
+    <a href="https://www.geeksforgeeks.org/what-is-wireframing/"
+    target="_blank">Read more</a>
+  </article>
+
+  <article class="card">
+    <img
+      class="articleimg"
+      src="https://blog.ronin.cloud/content/images/size/w1000/2022/01/git-feature.png"
+      alt="what is a branch in git?"
+    />
+    <h2>What is a branch in Git?</h2>
+    <p>
+      Branching is a powerful concept in Git that can feel confusing at first. In this comprehensive 2800+ word guide, we‘ll demystify Git branches to help you master this essential skill.
+    </p>
+    <a
```

Comment:
Please, don't split one `a` tag into separate lines. Keep it all in one line for better readability.